### PR TITLE
Dashboard and Metrics v14

### PR DESCRIPTION
**Pull request by nateh777**
##### What does the change add or fix?

Update dashboards and metrics migration for .NET 4.7.2 and new Rock common helpers.

---------

##### What does the change add or fix in a succinct statement that will be read by clients?

Update dashboards and metrics migration for .NET 4.7.2 and new Rock common helpers

---------

##### Who reported, requested, or paid for the change?

Warranty

---------

##### Does this update or add options to the block UI?

Proof it ran migration on dev environment:    
image

---------

##### What files does it affect?

- rocks.kfs.DashboardsAndMetrics/rocks.kfs.DashboardsAndMetrics.csproj
- KFSRock.sln.kfs

---------

##### Is it a breaking change for other versions/clients?

No

### CODE REVIEW

File: KFSRock.sln.kfs [hunk 1]
```diff
@@ -165,8 +166,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.Workflow.Action.P
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.Workflow.Action.WorkflowAttributes", "KFSRockAssemblies\rocks.kfs.Workflow.Action.WorkflowAttributes\rocks.kfs.Workflow.Action.WorkflowAttributes.csproj", "{9AC85805-0C97-424C-9506-DA6B281DB0CD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.WorkflowFromEWS", "KFSRockAssemblies\rocks.kfs.WorkflowFromEWS\rocks.kfs.WorkflowFromEWS.csproj", "{EEFCBE3D-6F6C-4C9B-8A19-B9FA04B4D6BF}"
```
Comment (mattbaylor) [inline KFSRock.sln.kfs:168]: Is this being removed intentionally? Are we not supporting it anymore?

File: KFSRock.sln.kfs [hunk 2]
```diff
@@ -177,8 +176,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.Workflow.Action.C
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.StepsToCare", "KFSRockAssemblies\rocks.kfs.StepsToCare\rocks.kfs.StepsToCare.csproj", "{44B89D21-5AF4-4E89-A49D-31B60EC4AB4D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.Shortcodes", "KFSRockAssemblies\rocks.kfs.Shortcodes\rocks.kfs.Shortcodes.csproj", "{EA104634-0BAA-402F-B901-E2016990237E}"
```
Comment (mattbaylor) [inline KFSRock.sln.kfs:180]: Same question with this one...

File: KFSRock.sln.kfs [hunk 3]
```diff
@@ -197,6 +194,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.Edify", "KFSRockA
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.SimpleTexting", "KFSRockAssemblies\rocks.kfs.SimpleTexting\rocks.kfs.SimpleTexting.csproj", "{63F45FC6-C6C8-4D27-8C5C-67C7563E85A6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.DashboardsAndMetrics", "KFSRockAssemblies\rocks.kfs.DashboardsAndMetrics\rocks.kfs.DashboardsAndMetrics.csproj", "{A9533BAE-C7B6-4222-9120-6EB49511F8D1}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "rocks.kfs.Microsoft365Utilities", "KFSRockAssemblies\rocks.kfs.Microsoft365Utilities\rocks.kfs.Microsoft365Utilities.csproj", "{EEFCBE3D-6F6C-4C9B-8A19-B9FA04B4D6BF}"
```
Comment (mattbaylor) [inline KFSRock.sln.kfs:199]: I'm guessing this replaced the EWS shortcake project, right?

Comment (nateh777) [reply KFSRock.sln.kfs:199]: Yes, we combined those two other projects with this single project, we just forgot to update the SLN in repo.

**Review by mattbaylor — APPROVED** [review]

**Pull request merged**